test/tools/router_check/test/config/ClusterHeader.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
If the docs mention supporting both yaml and json router configs, should we have a test case for json input?